jenkins/Jenkinsfile

```diff
@@ -34,20 +34,24 @@ pipeline {
         container('node'){
           dir("${env.WORKSPACE}/${env.PROJECT}") {
             echo "Install dependencies"
+            sh 'npm i -g npm@8' // seems like with older versions from 8.x branch workspaces doesn't work properly
```

Comment:
If you have any ideas how to properly fix it, I'm open for suggestions.

This solution is not perfect as I'm getting warnings whenever `npm` script is called:
```
[2022-10-04T12:45:17.298Z] npm WARN logfile Error: EACCES: permission denied, scandir '/root/.npm/_logs'
[2022-10-04T12:45:17.298Z] npm WARN logfile  error cleaning log files [Error: EACCES: permission denied, scandir '/root/.npm/_logs'] {
[2022-10-04T12:45:17.298Z] npm WARN logfile   errno: -13,
[2022-10-04T12:45:17.298Z] npm WARN logfile   code: 'EACCES',
[2022-10-04T12:45:17.298Z] npm WARN logfile   syscall: 'scandir',
[2022-10-04T12:45:17.298Z] npm WARN logfile   path: '/root/.npm/_logs'
[2022-10-04T12:45:17.298Z] npm WARN logfile }
```

so either in this or in the second PR, I'd like to fix it.